### ISSUE TITLE
crash on display of isogeny class of elliptic curve over Q(i)

### ISSUE THREAD

**Issue by JohnCremona** [LMFDB/lmfdb#790]
Go to http://www.lmfdb.org/EllipticCurve/2.0.4.1/%5B225%2C0%2C15%5D/a/6  (which is the base cghange to Q(i) of 15a1 (a.k.a. 15.a5) and click on the link to its isogeny class in the Related Objects column.  Boom.

I actually wanted to see the isogeny graph for something I was working on!

**Comment by JohnCremona** [comment]
I have fixed this; there were two different things to do:
   1. conductor labels over imaginary quadratic fields were designed by me but I really hate them and they look like [225,0,15].  The character "[", ",", "]"  need to be quoted using urllib in some places (and unquoted again in others.
   2. Then the isogeny graph I was interested in looking at was a shape which cannot occur over Q and that broke the special graph code.

I have fixed both these and will put in a pull request after more testing.
